### PR TITLE
feat(ImageBlock): fix ux review

### PR DESCRIPTION
**Pull request by celine-s**
tickets: [image block general](https://app.clickup.com/t/86783gd9e) and [accessibility](https://app.clickup.com/t/86782x7u0)

* Attachment button same size as download button
* Buttons now always stays on top of image (despite it's padding or image size)
* Focusing order corrected

### CODE REVIEW

**Review by fulopdaniel — APPROVED** [review]

File: packages/image-block/src/components/helpers.ts [hunk 1]
```diff
@@ -74,6 +74,19 @@ export const getCaptionPlugins = (appBridge: AppBridgeBlock) =>
             new ResetFormattingPlugin(),
         ]);
 
+export const getTotalImagePadding = (blockSettings: Settings): CSSProperties => {
```
Comment (ragi96) [inline packages/image-block/src/components/helpers.ts:77]: Add a unit test for that function

Comment (celine-s) [reply packages/image-block/src/components/helpers.ts:77]: We don't have that package within this block. I just added 3 tests to the ImageBlock.spec.ct.tsx (and fixed already a bug :D)

Comment (ragi96) [reply packages/image-block/src/components/helpers.ts:77]: For me this is also fine

**Review by ragi96 — APPROVED** [review]
Thanks

**Pull request merged**